src/blockchain/Params.cpp

```diff
@@ -818,7 +818,7 @@ auto Chains() noexcept -> const ChainData&
              false,
              0,
              opentxs::UnitType::TnCspr,
-             Bip44Type::TESTNET,
+             Bip44Type::CSPR,
```

Comment:
I have some doubts if we simply can do this.
Have all tests passed especially Test_BlockchainApi will fail?
I see that not. We used this TESTNET info to create address from Bip44.